stories/Icons.js

```diff
@@ -1,90 +1,69 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
-import {
-  AddPeopleIcon,
-  AddToBoardIcon,
-  AllItemsIcon,
-  ArrowIcon,
-  BellIcon,
-  BinocularsIcon,
-  BoardIcon,
-  CameraIcon,
-  CheckIcon,
-  ChevronLeftIcon,
-  ClipboardIcon,
-  CloneCompleteIcon,
-  CloudUploadIcon,
-  CloudUploadBoldIcon,
-  CommentIcon,
-  CopyIcon,
-  CreateBoardIcon,
-  CrossIcon,
-  DestructIcon,
-  DetachIcon,
-  DisableIcon,
-  DotsIcon,
-  DownIcon,
-  DownloadIcon,
-  DropdownIcon,
-  DropdownArrowIcon,
-  EnableIcon,
-  ErrorIcon,
-  ExpireTimeIcon,
-  FolderIcon,
-  GearIcon,
-  ImageIcon,
-  InfoIcon,
-  KeyIcon,
-  LogoutIcon,
-  MailIcon,
-  MoveIcon,
-  NewWindowIcon,
-  NotesIcon,
-  NotesBoldIcon,
-  OthersIcon,
-  PadlockIcon,
-  PadlockBoldIcon,
-  PadlockLockIcon,
-  PadlockLockOutlineIcon,
-  PadlockUnlockIcon,
-  PadlockUnlockOutlineIcon,
-  PenIcon,
-  PrivateIcon,
-  ProfileIcon,
-  PublicIcon,
-  PublicFolderIcon,
-  RemoveTagIcon,
-  SearchIcon,
-  SearchBoldIcon,
-  SendIcon,
-  ShareIcon,
-  SharedIcon,
-  StarIcon,
-  TagFilledIcon,
-  TagsIcon,
-  TeamIcon,
-  TrashBinIcon,
-  UpIcon,
-  UploadIcon,
-  UrlIcon,
-  VerticalDotsIcon,
-  ViewGridIcon,
-  ViewListIcon,
-  WalletIcon,
-  WarningIcon,
-  WatchFolderIcon,
-  ZoomCloseIcon,
-  ZoomInIcon,
-  ZoomOutIcon,
-  ZoomResetIcon,
-} from '../src/components/icons';
+import defaultTheme from '../src/themes/DefaultTheme';
+import * as icons from '../src/components/icons';
+
+const iconsArray = Object.keys(icons).map((key) => icons[key]);
+
+function getIcon(Icon) {
+  var _this = <Icon />;
```

Comment:
We don't use `var`s anymore and we don't use variable names starting with the underscore.
Also I would change that `this` into something like `iconComponent` ;)